### PR TITLE
Add more metrics about PageStorage

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflash/issues/6162

Problem Summary:

#### System table changes

* Add `max_pack_count_in_delta` to `system.dt_tables`
* Remove outdated columns for PageStorage V2 in `system.dt_tables`

#### Grafana changes
http://172.16.5.85:21540/d/qa-OjKNVk/test-cluster-tiflash-summary?orgId=1&from=1666781469235&to=1666803740418

* Rename "MVCC Snapshots" -> "Read Snapshots"
* Rename "StoragePool" -> "Page"
* Show log-files disk usage and log-files number
* Add metrics about P99 of write duration
* Breakdown the GC duration into a smaller granularity
* Move the Page GC panels from "Storage" to "Page"
* Show the number of pages in memory

image
image

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/Page/V3/PageStorageImpl.cpp [hunk 1]
```diff
@@ -413,21 +421,28 @@ PageStorageImpl::GCTimeStatistics PageStorageImpl::doGC(const WriteLimiterPtr &
     {
         GET_METRIC(tiflash_storage_page_gc_count, type_v3_mvcc_dumped).Increment();
     }
-    statistics.dump_snapshots_ms = gc_watch.elapsedMillisecondsFromLastTime();
+    statistics.compact_wal_ms = gc_watch.elapsedMillisecondsFromLastTime();
+    GET_METRIC(tiflash_storage_page_gc_duration_seconds, type_compact_wal).Observe(statistics.compact_directory_ms / 1000.0);
```
Comment (JaySon-Huang) [inline dbms/src/Storages/Page/V3/PageStorageImpl.cpp:425]:
```suggestion
    GET_METRIC(tiflash_storage_page_gc_duration_seconds, type_compact_wal).Observe(statistics. compact_wal_ms / 1000.0);
```

**Comment by JaySon-Huang** [comment]
Need update the tiflash-summary.json

**Comment by JaySon-Huang** [comment]
Working: the avg and max column files count per segment in the delta layer

**Review by lidezhu — APPROVED** [review]
LGTM

**Review by breezewish — APPROVED** [review]

File: dbms/src/Common/TiFlashMetrics.h [hunk 2]
```diff
@@ -149,17 +149,31 @@ namespace DB
         F(type_flush, {{"type", "flush"}}, ExpBuckets{0.001, 2, 20}),                                                                     \
         F(type_split, {{"type", "split"}}, ExpBuckets{0.001, 2, 20}))                                                                     \
     M(tiflash_storage_page_gc_count, "Total number of page's gc execution.", Counter,                                                     \
-        F(type_exec, {"type", "exec"}),                                                                                                   \
-        F(type_low_write, {"type", "low_write"}),                                                                                         \
+        F(type_v2, {"type", "exec"}),                                                                                                     \
```
[the remark this answers is not in the document]
Comment (JaySon-Huang) [reply dbms/src/Common/TiFlashMetrics.h]: I thought we could get a smoother line before and after the upgrade. But changing the label value seems more reasonable.
Changed.

File: metrics/grafana/tiflash_summary.json [hunk 3]
```diff
@@ -9911,7 +10095,7 @@
       "type": "row"
     }
   ],
-  "refresh": "30s",
+  "refresh": "10s",
```
Comment (JaySon-Huang) [inline metrics/grafana/tiflash_summary.json:10098]:
```suggestion
  "refresh": "30s",
```

**Pull request merged**